pymchelper/readers/fluka.py

```diff
@@ -357,3 +370,18 @@ def get_unit_for_scoring(cls, scoring: str) -> str:
 
         # if unknown scoring, return empty string
         return ''
+
+    @staticmethod
+    def get_axes_description(binning_type: int) -> AxesDescription:
+        """Get axes descriptions for binding type"""
+        if binning_type in (1, 11):  # cylindrical mesh
+            return AxesDescription(
+                AxisDescription("Radius (R)", "cm"),
+                AxisDescription("Angle (PHI)", "rad"),
+                AxisDescription("Position (Z)", "cm"))
+
+        # by default (which includes binning_type 0 and 10 we have Cartesian mesh)
```

Comment:
```suggestion
        # by default (which includes binning_type 1..10) we have Cartesian mesh
```

Reply:
WHAT 1 can have values from <0...8> and <10..18>.
Cartesian mesh is 0 and 10.
Maybe it should be worded differently.
Something like:

```
# As a default value for not yet implemented binning types.
# We use ones for Cartesian mesh
```

Reply:
OK, you are right, use something as you suggested below.